### DISCUSSION TITLE
Data not updating

### DISCUSSION THREAD

**Discussion by Zainabab00** [microsoft/finops-toolkit#872] [Q&A] [answered]
Hi, 

I have downloaded the FinOps hub about 1 month ago. The data have been up to date, with perhaps 1 to 2 days behind. Now I have issues with the data not updating,. Updated it this morning and the data was updated from 22.07 to 23.07. Today is 05.08. 
Does anyone have any suggestions?

Edit: 
Saw that the datatrigger for msexports have not been updated since 23.07. Is it any way to trigger it manually?

**Comment by flanakin** [comment, marked as answer]
A few thoughts:
1. Make sure the trigger is `Started`. I'm assuming it is since you checked the history, but just want to confirm.
2. Check the `msexports` container in storage to see if data is being exported from Cost Management. I suspect it hasn't.
3. Check the export in Cost Management to see if it still exists, is in the `Active` state, and doesn't have any errors in the run history.
